0009-lorawan-devaddr-routing.md

```diff
@@ -0,0 +1,173 @@
+- Start Date: 2020-02-20
+- HIP PR: <!-- leave this empty -->
+- Tracking Issue: <!-- leave this empty -->
+
+# Summary
+[summary]: #summary
+
+This HIP proposes a routing scheme for LoRaWAN packets post-join, via DevAddr. A
+block of DevAddr addresses will be obtainable for an OUI via a blockchain
+transaction. This range of addresses will be assigned to devices joined to this
+OUI and hotspots will be able to do fast and simple 'range routing' to route
```

Comment:
Can we explicitly define what exactly "range routing" is please. Maybe a glossary at the bottom of the doc would be useful?